packages/cozy-client/src/queries/dsl.js

```diff
@@ -78,6 +80,28 @@ class QueryDefinition {
     }
   }
 
+  /**
+   * Check the selector predicates.
```

Comment:
s/Check/Checks

EDIT: Saw that it was at least coherent with checkSortOrder.

Nit: When reading checkSomething(), I tend to think that the result will be boolean or that the method will throw. Here we just warn in case of a problem. I checked a bit my thesaurus and could find a better verb unfortunately.

Reply:
I see your point, but I guess we can rely on the jsdoc here